rosa_learning/deploying_application_workshop/learning-deploying-s2i-webhook-cicd.adoc

```diff
@@ -1,5 +1,5 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="cloud-experts-deploying-s2i-webhook-cicd"]
+[id="learning-deploying-s2i-webhook-cicd"]
 = Using Source-to-Image (S2I) webhooks for automated deployment
 include::_attributes/attributes-openshift-dedicated.adoc[]
 :context: cloud-experts-deploying-s2i-webhook-cicd
```

Comment:
I think you need a `+` on line 95:

![image](https://github.com/user-attachments/assets/6d9fd46c-3934-4f1c-a60e-17b8ea06eecd)